resources/amd-components/commons/d3ComponentBase.js

```diff
@@ -0,0 +1,60 @@
+
+/*
+ * d3 component, using the chartComponent
+ */
+define([
+  'cdf/components/ChartComponent',
+	'amd!cdf/lib/underscore',
+  './lib/d3'	
```

Comment:
This doesn't work because this relative path doesn't exist (`(...)/amd-components/commons/lib/d3.js`), the lib file is in path `(...)/components/commons/lib/d3.js`.
To fix this you need to either:

A) duplicate the file, copying it into the `(...)/amd-components/commons/lib/` path

B) configure RequireJS to have a path such as:
`requirePaths['d3ComponentLibrary/lib'] = '../resources/lib';`
that points to the root lib folder shared between amd and non-amd versions.

If you go with option B) then change the relative path above to the new one, or use the absolute path (such as `d3ComponentLibrary/lib/d3`).
You might have skipped this because of an invalid .gitignore configuration (it's incorrectly ignoring all lib folders), so on your local machine if you copied it into such folder it wasn't included in the PR commit.